### ISSUE TITLE
Formation/Conversion Wand doesn't work with Chisel blocks 1.12.2

### ISSUE THREAD

**Issue by EndermanParty** [HellFirePvP/AstralSorcery#800]
This is purely an enhancement but personally, I adore the building wands that the mod offers but I am an awful builder. I use the Chisel Mod blocks to build with since it makes my horrendous building look decent. 
Whenever I try to select a chiseled block, it sets the block type to that of the first chiseled block of that type. For instance, I am building my base out of Chiseled stone blocks and when I select either type of stone block, it sets the type to "Chiseled Stone- Mosaic". Chiseled Cobblestone blocks come out as the Chiseled Cobblestone Bricks. 
TLDR - I would love for Chisel integration with the formation and conversion wands.
Thanks!

**Comment by tofof** [comment]
Why was this closed? I came here to open exactly this issue, but it's unclear why the OP opened and closed it in the same day.

**Comment by Doomgull** [comment]
Is this an issue in the present 1.9.4 release version where the matching of the block is not correct, or is this a request for more compatibility with changing blocks? If the former, needs a new bug opened for review, with examples/screenshots. If this is a compatibility request, additional mod compat is usually denied due to existing feature expansion taking priority over adding compatibility which could be handled via an add-on mod.
